manifest

```diff
@@ -180,6 +180,7 @@ export AUR_PACKAGES="\
 	libretro-prosystem-git \
 	libretro-stella2014-git \
 	libretro-virtualjaguar-git \
+	kanit-font \
```

Comment:
Is this change needed here? If the package name changed, please put it in a separate commit (and push it to master!)